source/core/slang-string.h

```diff
@@ -159,13 +163,32 @@ namespace Slang
             return endData - beginData;
         }
 
-        bool operator==(UnownedStringSlice const& other)
+        int indexOf(char c) const
```

Comment:
As a convention thing, we are using `Int` and `UInt` for our default integer types, since they map to `intptr_t` and `uintptr_t` and so will always be pointer-sized, while `int` will be 32-bit for most 64-bit Windows compilers.

Not a big deal, so not asking for any change for now.

Reply:
I've seen the use of Int and UInt. I think there is an argument - if you are going to use them to map to int and unsigned int. That those types are that size for a variety of reasons - some historic. But there are speed and space advantages to using int/unsigned int. That if you are using a List to hold > 2^31 items - well is a List really the structure you should be using?